### ISSUE TITLE
demo.py got an error about npz

### ISSUE THREAD

**Issue by SabraHashemi** [zgojcic/3DSmoothNet#54]
fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: './data/demo/32_dim/cloud_bin_0.ply_0.150000_16_1.750000_3DSmoothNet.npz'

i ran demo.py and got this error

**Comment by Alexab** [comment]
I have the same problem

**Comment by zgojcic** [comment]
Did you check if the file is actually generated by the input parameterization code? Maybe you can post the whole output here

**Comment by Alexab** [comment]
I found the problem described above to be just a consequence. The reason was the following:
1) If python 2 and 3 are installed on the computer at the same time, then the scripts called from demo.py start with python 2 and crash.
2) After fixing the problem 1), I found that the method does not work with the new version of open3d. I fixed it by updating the demo.py code.

P.S. I think it is also possible to easily fix the problem of running with tf 2 by importing it in compatibility mode. But haven't tested it yet.
